sdk/communication/communication-administration/src/communicationIdentity/constants.ts

```diff
@@ -1,6 +1,4 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { apiVersion } from "./generated/src/models/parameters";
-
-export const SDK_VERSION: string = apiVersion.mapper.defaultValue;
+export const SDK_VERSION: string = "1.0.0-beta.3";
```

Comment:
What is the logic of the other repos to hardcode the SDK version?

This feels trappy where updating the SDK version requires changes in an unknown number of places.

Reply:
It's not great but the other JS libraries hard-code too.

There are at least 3 changes to make with a version bump: Changelog, package-version for autorest, and SDK_VERSION here.